keras_core/ops/numpy.py

```diff
@@ -256,6 +273,31 @@ def compute_output_spec(self, x1, x2):
 
 @keras_core_export(["keras_core.ops.add", "keras_core.ops.numpy.add"])
 def add(x1, x2):
+    """Add arguments element-wise.
+
+    Args:
+        x1, x2: Input tensors.
+
+    Returns:
+        The tensor containing the element-wise sum of `x1` and `x2`.
+
+    Examples:
+
+    >>> x1 = keras_core.ops.convert_to_tensor([1, 4])
+    >>> x2 = keras_core.ops.convert_to_tensor([5, 6])
+    >>> keras_core.ops.numpy.add(x1, x2)
+    Array([ 6, 10], dtype=int32)
```

Comment:
Use lowercase `array`

Reply:
When using JAX as backend, it prints `Array` with a capital `A`.

Happy to switch to lowercase, but for my understanding, what's the reasoning behind it?

<img width="365" alt="image" src="https://github.com/keras-team/keras-core/assets/104828547/6e211d68-06fa-48af-9ead-5861ab6d50ce">

Reply:
We want the outputs to be backend-agnostic, e.g. not `tf.Tensor`, etc. `array` is just the generic descriptor we've been using, that's all. It mirrors NumPy.

Reply:
👍